cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/brokers/CloudletToVmMappingBestFit.java

```diff
@@ -61,17 +60,12 @@
  * changing the method used for that goal,
  * without requiring to create a subclass to accomplish that.</p>
  *
- * <p>The example defines a the method {@link #bestFitCloudletToVmMapper(Cloudlet)} that
- * receives a Cloudlet a selects the best VM that fits it. That is,
- * the VM with lower capacity that is able to host that Cloudlet.
- * This method is given as parameter to the {@link DatacenterBroker#setVmMapper(Function)}
- * using Java 8 Functional Programming features.
- * This way, it changes the implementation of the VM selection policy
- * to the method implemented in this example.
+ * <p>The example uses the class {@link DatacenterBrokerBestFit} that
+ * an optimal mapping of Cloudlets to VMs based on the free Pes in VMs.
  * </p>
  *
  * @author Manoel Campos da Silva Filho
- * @since CloudSim Plus 1.3.0
+ * @since CloudSim Plus 4.3.8
```

Comment:
Maybe I changed the version incorrectly. This class is really from CloudSim Plus 1.3.0

Reply:
Yup sorry, was a mistake.